service/src/test/java/com/epam/esm/service/impl/CertificateServiceImplTest.java

```diff
@@ -30,132 +32,123 @@
     @Mock
     private CertificateDao certificateDao;
     @Mock
-    private CertificateDTOMapper certificateDTOMapper;
+    private ModelMapper modelMapper;
     @Mock
-    private TagService tagService;
-    @Mock
-    private SearchQueryProvider queryProvider;
-    @Mock
-    private ParametersProvider queryParametersProvider;
+    private TagDao tagDao;
     @InjectMocks
     private CertificateServiceImpl certificateService;
 
-    private static final String QUERY_PART = "query stub";
-    private static final String PARAMETER_STUB = "parameter stub";
-    private static final Integer RECORDS_AFFECTED_ONE = 1;
-    private static final Integer RECORDS_AFFECTED_NONE = 0;
-    private static final Long RECORD_ID = 1L;
+    private static final String TAG_NAME = "tag name";
+    private static final Integer VALUE_PAGE = 1;
+    private static final Integer VALUE_SIZE = 3;

```

Comment:
maybe size on page? or smth else that could help better understand what this parameter is

Reply:
sorry, renamed